### ISSUE TITLE
React Style unit inference does not work anymore

### ISSUE THREAD

**Issue by smirea** [felipethome/react-inline-transition-group#2]
When I wrap my components with a `<Transition />` they loose the automatic units that react adds (e.g. in React doing `style={{margin: 10}}` would automatically convert it to `style="margin: 10px"`, but the transition does not)

Example: if you inspect the children bellow, they have `style="margin: 10"` which is an invalid syntax. If you remove the `<Transition>` wrapper it works fine

```jsx
class TransitionTest extends Component {

    state = {
        list: '1234567890'.split(''),
    };

    render () {
        const {list} = this.state;

        return (
        <div>
            <Transition
                childrenBaseStyle={{
                    margin: 10,
                }}
            >
                {list.map(num =>
                    <div
                        key={num}
                        onClick={() => this.setState({list: list.filter(id => id !== num)})}
                    >{num}</div>
                )}
            </Transition>
        </div>
        );
    }

}
```

**Comment by felipethome** [comment]
@smirea Thanks for reporting. This feature probably never worked with this component and I never noticed because I always set my units instead of using the shortcuts.

When I was writing this component I didn't want to use any React internals since they could change and break my code. So, I do myself the style string based on the objects you pass to the component.

The problem is not difficult to solve and I will try to do it as soon as possible. In the meanwhile you can write your styles with the units.

**Comment by felipethome** [comment]
Fixed in version 1.1.0
